samples/bootloader/src/main.c

```diff
@@ -134,7 +136,7 @@ static void boot_from(u32_t *address)
 		nvic->ICPR[i] = 0xFFFFFFFF;
 	}
 
-	printk("Booting (0x%x).\r\n", (u32_t)address);
+	printk("Booting (0x%x).\r\n", fw_info->firmware_address);
 
 	uninit_used_peripherals();
```

Comment:
No changes needed in this function?

Reply:
Apparently not :). What are you missing?

Reply:
I was thinking uart deinitialization but I think I added that just saw the debug stuff moved around.